AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/consts/physics.py

```diff
@@ -0,0 +1,28 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Holds constants used across both hydra and non-hydra scripts.
+"""
+
+"""
+PhysX Components
+"""
+PHYSX_SHAPE_COLLIDER = "PhysX Shape Collider"
+
+"""
+Shape Components
+"""
+BOX_SHAPE_COMPONENT = "Box Shape"
+CAPSULE_SHAPE_COMPONENT = "Capsule Shape"
+CYLINDER_SHAPE_COMPONENT = "Cylinder Shape"
+POLYGON_PRISM_SHAPE_COMPONENT = "Polygon Prism Shape"
+QUAD_SHAPE_COMPONENT = "Quad Shape"
+SPHERE_SHAPE_COMPONENT = "Sphere Shape"
```

Comment:
question: Should these have the post-fix `COMPONENT` if it's not in the name? Or Maybe they should be called `..._SHAPE_COMPONENT_NAME` to be clearer?

Reply:
Most of these consts are on a deprecation path to be relocated to their associated component behavior class file. 

The reason I had COMPONENT in the name is due to the existence of the shape property in some components which could cause confusion if not clearly named.

However, you're right, the names can be clearer on intent and I've renamed tome to the second pattern.

Reply:
Since the properties are found in `azlmbr.physics.property`, I'm going to omit the COMPONENT from the const names for components, and leave a note in the comment to use the above for shape properties.